libraries/SITL/SIM_last_letter.cpp

```diff
@@ -53,39 +53,24 @@ last_letter::last_letter(const char *home_str, const char *_frame_str) :
 void last_letter::start_last_letter(void)
 {
     pid_t child_pid = fork();
-    if (child_pid != 0) {
-        return;
+    if (child_pid == 0) {
+      // in child
+      close(0);
+      open("/dev/null", O_RDONLY);
+      for (uint8_t i=3; i<100; i++) {
+          close(i);
+      }
+      int ret = execlp("roslaunch",
```

Comment:
Followed the Gazebo SITL code to re-form the calling argument to comply with the new Gazebo simulator calling line of last_letter. This is expected to be expanded with more functionality, such as the HOME argument in the future.